etc/distrepos.conf

```diff
@@ -104,6 +104,19 @@ mirror_hosts =
 # Parent directory within dest_root for tarball client
 tarball_install = tarball-install
 
+#
+# Release series
+#
+# Each release series comprises a series of tagsets, as well as an osg-release <noarch>
+# rpm containing yum repository definitions for the rpms contained in each tagset
+
+[series 24-main]
+dest = osg/24-main
+dvers = el8 el9
+
+[series 23-main]
+dest = osg/23-main
+dvers = el8 el9
 
```

Comment:
This feels a little incorrect to add, especially since series are not directly tied to their tags. However, the "dest" dir of every tag is several directories deep within the series and creating symlinks at `../../../latest-release.rpm` felt even less correct